### PR TITLE
DOC-2414: TinyMCE 7.1.1 Release Documentation

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2414

Site: [Release Notes](http://docs-release-711.staging.tiny.cloud/docs/tinymce/latest/release-notes)
Site: [Changelog](http://docs-release-711.staging.tiny.cloud/docs/tinymce/latest/changelog)

Changes:
* TinyMCE 7.1.1 Release Notes & Changelog

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [-] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+In {productname} {release-version}, this issue has been addressed. Now, the focus remains inside the `Insert/Edit Image` dialog after uploading an image, allowing users to navigate within the dialog using the keyboard as expected.
+
+=== Deleting into a list from a paragraph that has an `img` tag could cause extra inline styles to be added.
+// #TINY-10892
+
+Previously in {productname}, there was an issue with lists when the host browser was Google Chrome. The issue occurred when there was content with a list followed by a paragraph that contained an `img` element. If the cursor was placed at the start of the paragraph and the backspace key was pressed, the content of the paragraph would be merged into the list item, but the `img` element and other neighboring elements would get undesired inline font styles.
+
+This issue has been fixed in {productname} {release-version}. Now, a check has been added to prevent the inline styles from being added following this action. As a result, the resulting HTML no longer gets undesired inline styles, and the behavior is as expected.
+
+=== Resolved an issue where emojis were not loading correctly due to a broken CDN.
```
Comment (kemister85) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:93]:
```suggestion
=== Resolved an issue where emojis configured with the `emojiimages` database were not loading correctly due to a broken CDN.
```

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -4,66 +4,82 @@
...
+== 7.1.1 - 2024-05-22
 
-### Added
+=== Fixed
+
+* Insert/Edit image dialog lost focus after the image upload completed.
+// #TINY-10885
+* Deleting into a list from a paragraph that has an `img` tag could cause extra inline styles to be added.
+// #TINY-10892
+* Resolved an issue where emojis were not loading correctly due to a broken CDN.
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:15]:
```suggestion
* Resolved an issue where emojis configured with the `emojiimages` database were not loading correctly due to a broken CDN.
```

**Review by ArvinJ-H — APPROVED** [review]

**Review by kemister85 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc]: resolved in `feature` commit.

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: resolved in `feature` commit.

**Review by MitchC1999 — APPROVED** [review]

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Review by tiny-ben-tran — APPROVED** [review]

**Pull request merged**